components/Profile/EditProfileModal.tsx

```diff
@@ -6,18 +6,31 @@ import axios from "../../util/axios"
 import { options } from "./SkillOptions"
 import { useAuth } from "../../context/auth"
 
-const EditProfile = ({
+type ModalPropTypes = {
+    handle:string,
+    bio:string,
+    interest:string,
+    name:string,
+    url:string,
+    username:string,
+    handleClose:()=>void,
+    show:boolean,
+    closable:boolean,
+}
+
+const EditProfile :React.FC<ModalPropTypes> = ({
     handle,
     bio,
     interest,
     name,
-    url,
     username,
     handleClose,
     show,
     closable,
 }) => {
-    const { firebaseUser } = useAuth()
+
+
+    const { firebaseUser } :any = useAuth()
```

Comment:
Not required after #102.

Reply:
Will change it as soon as #102 is merged.

Reply:
Okay